python/beeai_framework/workflows/workflow.py

```diff
@@ -61,12 +79,23 @@ class Workflow(Generic[T, K]):
 
     _RESERVED_STEP_NAMES: ClassVar = [START, SELF, PREV, NEXT, END]
 
+    emitter: Emitter
```

Comment:
not needed as you are defining in the constructor

Reply:
https://github.com/i-am-bee/beeai-framework/pull/416/commits/f8b1d1a685b116c696561dd117538be6e52d1949